kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/TlsSecretsTest.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.kubernetes.operator;
+
+import java.nio.file.Path;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import io.kroxylicious.kubernetes.api.v1alpha1.KafkaProxyBuilder;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class TlsSecretsTest {
+
+    private static final String LISTENER_NAME = "my-listener";
+    private static final String LISTENER_NAME2 = "another-listener";
+    private static final String SECRET_NAME = "certs";
+    private static final String SECRET_NAME2 = "another-secret";
+
+    @Test
+    public void singleListener() {
+        var proxy = new KafkaProxyBuilder()
+                .withNewSpec()
```

Comment:
my precious formatting was lost :cry:

Reply:
Did you try wrapping with `// @formatter:off` and `// @formatter:on`?